crates/ruff_python_formatter/tests/snapshots/format@expression__call.py.snap

```diff
@@ -386,10 +386,10 @@ threshold_date = datetime.datetime.now() - datetime.timedelta(  # comment
 )
 
 # Parenthesized and opening-parenthesis comments
-func(x for x in y)
+func((x for x in y))
 
 func(  # outer comment
-    x for x in y
+    (x for x in y)
```

Comment:
when it breaks this looks fine, but i kinda prefer no parentheses for consistency